packages/file-type-icons/src/FileIconType.ts

```diff
@@ -18,6 +18,7 @@ export enum FileIconType {
   documentsFolder = 10,
   picturesFolder = 11,
   linkedFolder = 12,
+  list = 13,
 }
 
-export type FileIconTypeInput = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+export type FileIconTypeInput = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13;
```

Comment:
Please change this to `keyof typeof FileIconType`.

Reply:
replace the entire line with that? this gave me a compile error until i added the 13, and it seems related to this test case but im frankly not sure what its other purpose is

https://github.com/microsoft/fluentui/blob/ee40f3275316d43f9ec10ec3cf11ed9c4d3e64f9/packages/file-type-icons/src/FileIconType.test.ts#L5